src/components/InvertedFlatList/BaseInvertedFlatList.js

```diff
@@ -155,6 +179,7 @@ class BaseInvertedFlatList extends Component {
                 windowSize={15}
                 removeClippedSubviews={this.props.shouldRemoveClippedSubviews}
                 maintainVisibleContentPosition={{minIndexForVisible: 0, autoscrollToTopThreshold: 0}}
+                onScrollToIndexFailed={() => {}}
```

Comment:
Would you mind explaining why this is necessary? Thanks! (to me, not as a code comment)

Reply:
Yes its a react native thing. If i don't have this there will be a number of errors/warnings that pop up telling me i should have onScrollToIndexFailed defined.